project4/project/ebs/templates/ebs/home.html

```diff
@@ -322,116 +320,162 @@ <h4 class="fontfamily1">Fashion Week Parties: Night 4</h4></a>
 				<div class="col-md-4 ">
 					<!-- MOST POPULAR -->
 
-					<h4><strong>MOST POPULAR</strong></h4>
-					<hr class="hrcol">
+					<h4 class="popular-hr"><strong>MOST POPULAR</strong></h4>
```

Comment:
use different heading tags to show different titles. e.g. <h1>, <h2>, <h3>, <h4>, <h5>, <h6>